bessctl/module_tests/worker_split.py

```diff
@@ -63,7 +63,7 @@ def test_worker_split_default_mode(self):
             bess.reset_all()
 
     def test_worker_split_mask_mode(self):
-        NUM_WORKERS = 6
+        NUM_WORKERS = 2
```

Comment:
(Hopefully my individual comment on earlier commit doesn't vanish...)

I presume this is needed for Travis testing because of just 2 cores. Maybe we could limit it to ncpu rounded to even? (ncpu available by running `nproc` or importing `multiprocessing` and calling `multiprocessing.cpu_count()`)